src/Tool-SystemReporter/SizeReporter.class.st

```diff
@@ -0,0 +1,84 @@
+"
+A subclass of AbstractReporter that computes various metrics over the image
+"
+Class {
+	#name : #SizeReporter,
```

Comment:
Also here. What size means here? 
It reports source code statistics. And better would be to name it accordingly like SourceCodeSizeReporter

Reply:
agreed